jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This also seems IDE specific, should be git ignored.

Reply:
If we really want to, we can include them in a wiki help page about visual studio code specific configurations

Reply:
Technically it's `tsserver` specific, so any IDE using that (which would be most, if not all supporting TypeScript) should support this file.

That doesn't really address your point, but I'd support putting it in a wiki.